.github/workflows/test.yaml

```
@@ -34,7 +34,7 @@ permissions:
 jobs:
   test:
     name: ${{ matrix.type }} ${{ matrix.flavor }}
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.type == 'install' && 'uds-swf-ubuntu-big-boy-8-core' || 'ubuntu-latest' }}
```

```suggestion
    runs-on: 'ubuntu-latest'
```
If upgrade passes on `ubuntu-latest` it should work for install too (since that should be fewer resources)

---

Yeah, that makes sense. I think it was just my poor commits as it seems to run fine now :)